cluster/examples/vm-ephemeral.yaml

```diff
@@ -3,6 +3,8 @@ kind: VirtualMachine
 metadata:
   creationTimestamp: null
   name: vm-ephemeral
+  labels:
+    kubevirt.io/vm: vm-ephemeral
```

Comment:
with the new logic, VMs that don't have a label will not be exposed. therefore a label was added to the samples